### PR TITLE
request as fallback for xhr

### PR DESCRIPTION
**Pull request by fritzvd**
Hi Max,

I was trying to build a server side script. But it failed when I updated to 1.0.1.
I added a fallback to request which has a very similar api to xhr.

Fritz

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,5 +1,13 @@
 var querystring = require('querystring')
-var xhr = require('xhr')
+var xhr;
+
+// check if running in browser or as server
+try {
+  xhr =  require('xhr');
```
Comment (MaxGfeller) [inline index.js]: Could you replace the try/catch blog with a simple `if (xhr.open)`? Makes the whole thing smaller and more readable.

**Comment by MaxGfeller** [comment]
Awesome, thank you! I added a note to the code, could you change this? Oh and please make sure that this uses [JavaScript Standard Style](https://www.npmjs.com/package/standard) (just run `npm test` to check)

**Comment by fritzvd** [comment]
Hi Max,

Thanks for the reply, whoops and I see now that I was sloppy. Sorry for
that. Ran the test with the new commit and it's fine I tried doing that.
Also rectified the xhr.open() try to if (xhr.open).

Fritz


On 1 June 2015 at 13:53, Max Gfeller <notifications@github.com> wrote:

> Awesome, thank you! I added a note to the code, could you change this? Oh
> and please make sure that this uses JavaScript Standard Style
> <https://www.npmjs.com/package/standard> (just run npm test to check)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MaxGfeller/youtube-search/pull/13#issuecomment-107414792>
> .
>



-- 
*Nelen & Schuurmans *

 Postbus 1219  3500 BE Utrecht
Zakkendragershof 34-44 Utrecht
tel. 030 2330 200
@fritzvd <http://twitter.com/fritzvd>
www.nelen-schuurmans.nl

**Comment by MaxGfeller** [comment]
Alright, i simplified the whole thing a bit and pushed a new version to npm.

**Pull request merged**